jbt-it/neomdb
# #127 client implement traineesection

Created the table for internal projects, currently without API and created a grid for the members of the current trainee generation. Also prepared the grid for APIs, the code is currently outcommented

Please check, if table and grid is shown properly and for potential optimization of the code.
Also check, if checkboxes for IPs can be ticked, the filter function works and that the checkboxes for the trainees are disabled


How to test new version:
1. user without permission
- check if all ips are loaded
- check if all trainees for the respective ips are loaded
- check if you can switch between generations
- check if the ip detail page is linked, should navigate to /internes-projekt/:id (detail page is another issue #135 )
- check if the links to the trainee profile page is working
2. user with permission
- check if the button to add an ip is only for the most recent generation available
- check wether the errors/warnings pop up if you leave some fields blank when adding an ip
- check if the alert pops up after adding an ip
- check wether the table for the workshops is shown porperly
